### ISSUE TITLE
playing music 

### ISSUE THREAD

**Issue by amin1562** [Splamy/TS3AudioBot#964]
**Describe the bug**
well, I installed both platform and in new version that uploaded today in windows works fine, but in Ubuntu it's not.
When I request a play(!play https://dl.sevilmusic.com/cdn/music/svs/Emo%20Band%20-%20Na%20Nagoo%20Na%20%5BSevilMusic%5D.mp3?_=1) this works in windows but in Ubuntu it says Could not load. (media: Unknown request error.)

**Version**
Version: 0.13.0-alpha.39
Branch: develop
CommitHash: 588b5d096f97bd316d204e6d9569ac73b0ff33c2

**Platform**
Ubuntu 20.04.3 LTS

**Log**
```
2022-01-30 16:24:37.2951| INFO|Core|Setup.LogHeader [============ TS3AudioBot started =============]
2022-01-30 16:24:37.3864| INFO|Core|Setup.LogHeader [ Date: Sunday, 30 January 2022 16:24:37
2022-01-30 16:24:37.5726| INFO|Core|Setup.LogHeader [ Version: 0.13.0-alpha.39/develop/588b5d09
2022-01-30 16:24:37.5726| INFO|Core|Setup.LogHeader [ Build: Release
2022-01-30 16:24:37.5726| INFO|Core|Setup.LogHeader [ Platform: Ubuntu 20.04.3 LTS (Focal Fossa) (64bit)
2022-01-30 16:24:37.5726| INFO|Core|Setup.LogHeader [ Runtime: .NET (6.0.1) ServerGC:True GC:SustainedLowLatency
2022-01-30 16:24:37.5761| INFO|Core|Setup.LogHeader [ Opus: libopus 1.3.1 (x64)
2022-01-30 16:24:37.5761| INFO|Core|Setup.LogHeader [==============================================]
2022-01-30 16:24:38.8375| WARN|1|PacketHandler`2.FetchPackets Dropping not decryptable packet: 54 53 33 49 4E 49 54 31 00 00 A6 00 01 
2022-01-30 16:24:38.8381| WARN|0|PacketHandler`2.FetchPackets Dropping not decryptable packet: 54 53 33 49 4E 49 54 31 00 00 A6 00 01 
2022-01-30 16:24:38.8381| WARN|0|PacketHandler`2.FetchPackets Dropping not decryptable packet: C7 F9 20 17 D9 6A 97 52 00 01 26 13 EF 
2022-01-30 16:24:38.8381| WARN|1|PacketHandler`2.FetchPackets Dropping not decryptable packet: 66 C6 38 77 04 2F 1E D2 00 01 26 A2 85 
2022-01-30 16:24:38.9165| INFO|Core|Ts3Client.ConnectClient Client connected.
2022-01-30 16:24:38.9595| INFO|Core|Ts3Client.ConnectClient Client connected.
2022-01-30 16:24:54.0116| INFO|0|Bot.OnMessageReceived User A.M.I.N requested: !play [URL]https://dl.sevilmusic.com/cdn/music/svs/Emo%20Band%20-%20Na%20Nagoo%20Na%20%5BSevilMusic%5D.mp3?_=1[/URL]
```
in ts:
<19:54:52> "A.M.I.N": !play https://dl.sevilmusic.com/cdn/music/svs/Emo%20Band%20-%20Na%20Nagoo%20Na%20%5BSevilMusic%5D.mp3?_=1
<19:54:52> "TS3AudioBot1": Error: Could not load. (media: Unknown request error.)

**Comment by Splamy** [comment]
can you post the detailed log from the logfile, around the time your sent your request (19:54:52) ?

**Comment by amin1562** [comment]
Well,it's same.
Let me try again.
Here is the log for now
16:31:23.9356| INFO|4| User A.M.I.N requested: !play [URL]https://streams.ilovemusic.de/iloveradio1.mp3[/URL]
16:31:25.0600| INFO|Api| ::ffff:62.102.138.128 Requested: /api/bot/use/2/(/json/merge/(/bot/info)/(/info/%40-1/5)/(/song)/(/repeat)/(/random)/(/volume))
16:31:27.0598| INFO|Api| ::ffff:62.102.138.128 Requested: /api/bot/use/2/(/json/merge/(/bot/info)/(/info/%40-1/5)/(/song)/(/repeat)/(/random)/(/volume))
16:31:29.0695| INFO|Api| ::ffff:62.102.138.128 Requested: /api/bot/use/2/(/json/merge/(/bot/info)/(/info/%40-1/5)/(/song)/(/repeat)/(/random)/(/volume))
Time is incorrect on server, but it's for now.
And also in ts:
<20:01:23> "A.M.I.N": !play https://streams.ilovemusic.de/iloveradio1.mp3
<20:01:23> "TS3AudioBot5": Error: Could not load. (media: Unknown request error.)

(it's same for radio links, and also music links, most of the radio links after I send the request gives me the same error)

**Comment by amin1562** [comment]
Well, the version that I installed in windows platform is not as same as I installed in Ubuntu.

Current version in windows platform that works fine(playing music links and radio):

Version: 0.12.2
Branch: master
CommitHash: 4c6be03646d78cdbfe007ecc697596256ec23617

But I want to have this in Linux not windows, is there any way to fix this kinda bug?

**Comment by amin1562** [comment]
@Splamy can you help me to solve this?

**Comment by amin1562** [comment]
Issue solved.
I checked codes, logs, FFmpeg ... and I did not find any problem there.
Make sure to check mikrotik of the VPS that you're using, I could play radio but the only one with IP not with the link, I entered DNS but last for 1sec, and there I found out there is a issue in mikrotik and that was the answer.
